setup.py

```diff
@@ -44,7 +44,7 @@ def run(self):
 
 data = dict(
     name='python-datetime-tz',
-    version='0.5',
+    version='0.5.1',
```

Comment:
We probably need to be careful about putting an official-looking version number like this into our own repo and doing a private release...